contracts/contracts/rem.swap/src/system_info.cpp

```diff
@@ -0,0 +1,110 @@
+/**
+ *  @copyright defined in eos/LICENSE.txt
+ */
+
+#include <rem.swap/rem.swap.hpp>
+
+namespace eosio {
+
+    struct [[eosio::table("global"), eosio::contract("rem.system")]] eosio_global_state : eosio::blockchain_parameters {
```

Comment:
move structures to hpp files if possible

Reply:
This structures should not be in the `rem.swap.hpp` file.

Reply:
import this structure from system header file

Reply:
@Art17, done.